ContextMenu.Droid/ContextMenuViewRenderer.cs

```diff
@@ -149,6 +149,9 @@ private async void OnScrollToRequestedNew(object sender, ScrollToRequestedEventA
 				await Task.Delay(TimeSpan.FromMilliseconds(1));
 				cycle++;
 
+				if (CanHandle() == false)
```

Comment:
I would check this also before entering while loop

Reply:
Done